### PR TITLE
Update documentation

### LINKED ISSUE #753: When I run the themekit setup it creates a new theme in the same directory. Can this be routed into a subdirectory?
I followed the instructions as posted here:

https://shopify.github.io/themekit/

specifically these steps to Create a New Theme:

```
Create a new theme.
If you are starting from scratch and want to get a quick start, run the following:

theme new --password=[your-password] --store=[your-store.myshopify.com] --name=[theme name]
```

When I did this, it created the theme in the parent folder (I was expecting a sub-folder to be created). 

Now I'm not sure if I can copy-paste and move the specific files to a subfolder or what to do with the orphaned files, it's really inconvenient and sort of a barrier to a newbie getting started on this. 

I'd recommend either updating the documentation to include this step, or auto-creating a subfolder given the theme name.

### PR DESCRIPTION
**Pull request by Andrewngabriel**
I've updated the documentation for the `new` command to include a table with optional flags and indicate that by default the command will use the current directory and not create a new one.

Fixes #753, #795 

### Warn Checklist
- [ ] This changes the interface and requires a Major/Minor version change.
- [ ] I have :tophat:'d these changes by using the commands I changed by hand.
- [ ] I have added a dependancy to the project.
- [x] I have considered any potential impact on [node-themekit](https://github.com/Shopify/node-themekit)

### CODE REVIEW

**Comment by Andrewngabriel** [comment]
I've signed the CLA but it's still showing the error. Any reason why that's the case @tanema @chrisbutcher ? Can I re-run it?

**Comment by tanema** [comment]
Hey @Andrewngabriel I fixed the CLA bot. Can you also update the help in the command? https://github.com/Shopify/themekit/blob/master/cmd/new.go#L17-L22

**Comment by Andrewngabriel** [comment]
@tanema Just updated it!

**Comment by Andrewngabriel** [comment]
I saw issue #795  this morning and figured this PR is still open and it's related to documentation, I might as well update it.

File: cmd/new.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 
 var newCmd = &cobra.Command{
 	Use:   "new",
-	Short: "New will create theme using Shopify Timber in the same directory where it gets called from",
+	Short: "New will create theme using Shopify in the same directory where it gets called from",
```
Comment (tanema) [inline cmd/new.go:17]: Hi thanks for removing Timber, but this no longer makes sense. Here is a patch to fix it up.

```suggestion
	Short: "New will generate a new blank slate theme in the same directory where it gets called from and create a new theme on Shopify with those files.",
```

**Review by tanema — APPROVED** [review]
Thanks for the changes. Just one change

**Comment by Andrewngabriel** [comment]
Fixed

**Comment by tanema** [comment]
thanks again for your help!

File: cmd/new.go [hunk 2]
```diff
@@ -14,10 +14,12 @@ import (
 
 var newCmd = &cobra.Command{
 	Use:   "new",
-	Short: "New will create theme using Shopify Timber",
+	Short: "New will generate a new blank slate theme in the same directory where it gets called from and create a new theme on Shopify with those files.",
```
Comment (andyw8) [inline cmd/new.go:17]: @tanema I wonder if someone might confuse 'slate' with https://github.com/Shopify/slate ?

**Pull request merged**